magpie-core/pom.xml

```diff
@@ -55,6 +56,18 @@
       <scope>compile</scope>
     </dependency>
 
+    <!-- TODO: design - are we able to avoid wiring AWS into the core package? magpie-aws / magpie-dmap plugin? -->
```

Comment:
Remind me why this is required in `core`?

Reply:
We need this due to the AWS Clients invocation under the core module for Role/Policy creation and Lambda creation/invocation/deletion

Reply:
Makes sense.  At some point we should make a `dmap` module and move it to there, but we'll plan that for a future release.